docs/docusaurus/docs/cloud/expectation_suites/manage_expectation_suites.md

```diff
@@ -28,7 +28,7 @@ Automatically create an Expectation Suite that you can use to determine if your
 
 5. Click **Generate Expectations**. 
 
-    A new Expectation Suite appears in the Expectation Suites pane.
+    It can take several minutes to create the Expectation Suite. When the process is complete, a new Expectation Suite appears in the **Expectation Suites** pane.
```

Comment:
I'm afraid that several minutes sounds too intimidating. Maybe something like "It will take seconds or minutes to create the Expectation Suite based on your data..."

Reply:
@talagluck, unfortunately, that is the standard message when providing time estimates. If it takes less time, users are happy. If it takes longer, users understand, as we weren't specific.